src/libcore/slice/mod.rs

```diff
@@ -5773,6 +5774,34 @@ unsafe impl<'a, T> TrustedRandomAccess for RChunksExactMut<'a, T> {
 /// assert_eq!(slice[0], 42);
 /// ```
 ///
+/// ### Incorrect usage
+///
+/// The following `join_slices` function is **unsound** ⚠️
```

Comment:
I like the warning icon, we should add that to the `MaybeUninit` examples as well :)

Reply:
😄 https://emojipedia.org/warning/ 👌

Reply:
Do you want to make a PR to use this in the `MaybeUninit` docs or should I?

Reply:
I think it is best you do it: I'd put warning signs all over `MaybeUninit` docs so I'll let you do that in a more accurate manner 😅